app/ide-desktop/eslint.config.js

```diff
@@ -0,0 +1,354 @@
+/** @file ESLint configuration file. */
+/** NOTE: The "Experimental: Use Flat Config" option must be enabled. */
```

Comment:
Here is a note telling that something has to be enabled without explanation why it has toi be.